liquibase-service/src/main/resources/db/rdb_modern/tables/007-create_nrt_investigation-001.sql

```diff
@@ -115,11 +115,21 @@ CREATE TABLE dbo.nrt_investigation (
 );
 
 IF EXISTS (SELECT 1 FROM sysobjects WHERE name = 'nrt_investigation' and xtype = 'U')
-BEGIN
+    BEGIN
 
         IF NOT EXISTS(SELECT 1 FROM sys.columns WHERE name = N'investigation_form_cd' AND Object_ID = Object_ID(N'nrt_investigation'))
             BEGIN
             ALTER TABLE dbo.nrt_investigation ADD investigation_form_cd VARCHAR(50);
             END;
 
-END;
\ No newline at end of file
+    END;
+
+IF EXISTS (SELECT 1 FROM sysobjects WHERE name = 'nrt_investigation' and xtype = 'U')
```

Comment:
Is this second 'IF EXISTS' needed? The changing code can be placed inside the first block for better readability and reducing number of code lines.

Reply:
Updated, with thanks!